### ISSUE TITLE
Questions on the specification of FDP

### ISSUE THREAD

**Issue by HimmelStein** [openspending/fiscal-data-package#107]
1. a yes-no question:

Is the structure of this datapackage.json out of dated?
https://github.com/os-data/Beneficiaries-From-EAGF-and-the-EAFRD-in-Slovakia-2014/blob/master/datapackage.json

2. a why question:

In the 'Labels and Hierarchies' example provided by the official webpage, http://fiscal.dataprotocols.org/examples/labels-and-hierarchies,
in the 'mapping' part, the 'source' of 'amount' is 'expenditure'. but, there is no such field in the physical table. why?

"mapping": {
    "measures": {
      "amount": {       
        "source": "expenditure",

3. a how question:

in the same example above, the 'primaryKey' of 'classification' is a list of  "level1_id", "level2_id". However, here "level2_id" is enough, if we know High Education is a kind of Education. how is primary key identified in general? will or should such 'a kind of' relation be considered in 'basic fiscal concept', as well as in the terminology system (e.g. code list) of OpenBudget.EU ?

best regards and Merry X'mas!

**Comment by danfowler** [comment]
Hi @HimmelStein 

1. Yes, the datapackage is outdated.  I've updated it and moved it here: https://github.com/openspending/fiscal-data-package-demos/tree/master/eu-beneficiaries-from-eagf-and-the-eafrd-in-slovakia-2014 . I'll be migrating all outdated datasets from os-data to fiscal-data-package-demos as appropriate.
1. This was an error; thanks for pointing it out!  The source should be `amount`.  Fixed: http://fiscal.dataprotocols.org/examples/labels-and-hierarchies/
1. This is a very good point.  I'm going to do some thinking about how specifying the use of `primaryKey`, especially in relation to the new `parent` and `labelfor` keys. For now, I'm leaving it explicit in the example and will update this issue after some more thought.  Re: considering this a "basic fiscal concept", my inclination is no, but again, will update this issue after some more thought.

Happy Holidays!
